src/ngx_http_lua_module.c

```diff
@@ -93,6 +93,13 @@ static ngx_command_t ngx_http_lua_cmds[] = {
       0,
       NULL },
 
+    { ngx_string("lua_intercept_log"),
```

Comment:
I'm afraid this is confusing. NGINX already has two different kinds of logs: access logs and error logs. We should be more explicit here to avoid confusions.